docs/troubleshooting.md

```diff
@@ -19,6 +19,17 @@ If you attempted to deploy Wildebeest in your account and something failed, or y
 
 You can now start a clean install.
 
+### Finding error causes
+
+If you get an error page or 500 status error for the main HTML document you can find the source of the error using the wrangler CLI:
+
+- go to your Cloudflare account dashboard > Pages and click on `wildebeest-YOURNAME`
+- click on the Production deployment for your project (View details)
+- copy the second UUID in the URL, which is your deployment ID
+- get wrangler CLI if you don't already have it (`yarn global add wrangler`)
```

Comment:
`yarn` isn't the default node package manager, so I would go with `npm install -g wrangler` instead (you can also could include the `yarn` and `pnpm` versions as well if you think that that would be valuable)

Reply:
yarn is required by this project? (`yarn.lock`)

Reply:
ah... mh.... yeah you do have a point....  but people (like me! :grin:) can use yarn locally for the project and just use `npm` globally, anyways I've seen that you've changed it to refer to the cloudflare docs, great idea! :smiley: